Cohort1/Medlista/shared/src/commonMain/kotlin/com/example/medlista/Greeting.kt

```diff
@@ -6,4 +6,4 @@ class Greeting {
     fun greet(): String {
         return "Hello, ${platform.name}!"
     }
-}
\ No newline at end of file
+}
```

Comment:
not totally sure what happened here - took out the bracket and then replaced it?